packages/plugin-core/assets/dendron-ws/vault/welcomeV2.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Cat Coding</title>
+  </head>
+  <body>
+    <!-- <img src="https://media.giphy.com/media/JIX9t2j0ZTN9S/giphy.gif" width="300" /> -->
+    <img
+      src="https://foundation-prod-assetspublic53c57cce-8cpvgjldwysl.s3-us-west-2.amazonaws.com/assets/logo-256.png"
+      alt="Dendron Logo"
+    />
+
+    <p>Click the following button to see the function in action</p>
+    <input type="button" id="btn" value="Initialize Workspace" />
+
+    <script>
+      var btn = document.getElementById("btn");
+      btn.addEventListener("click", function () {
+        console.log("webview - btn click callback");
+        const vscode = acquireVsCodeApi();
+
+        vscode.postMessage({
```

Comment:
let me know if you want to change `initializeWorksace` to a file picker. think that makes sense